### ISSUE TITLE
Importing from same directory throws unnecessary errors

### ISSUE THREAD

**Issue by tjf801** [microsoft/pyright#661]
**Describe the bug**
I have two files in a directory, and when I try to import the other file, I get an error using pyright, but no error with the normal pylint extension.

Folder
  main.py
  levels.py

**To Reproduce**
main.py:
```
import levels
#do other stuff
```
levels.py:
```
VAR = 3
#the code does not really matter
```

**Expected behavior**
the expected behavior would be to successfully see that the levels.py file exists, and import it.

**Screenshots or Code**
https://imgur.com/a/dlXR70W

**VS Code extension or command-line**
I am running pyright as a VS Code extension, version 1.1.35

**Comment by erictraut** [comment]
You are using an absolute import (as opposed to a relative), so the import resolution logic will try to resolve the import using the configured import paths. Unless otherwise specified, pyright will assume that your import paths include 1) the root of your workspace, and 2) a subdirectory within your workspace called "src". If you would like it to look in other locations, you need to configure it appropriately. Apparently pylint is following python 2 import resolution rules and assuming this is a relative import. Pyright follows python 3 import resolution rules and will use relative imports only if you use the "import .levels" syntax.

So you have the following options:
1. Change your code to use a relative import instead of absolute.
2. Create a pyrightconfig.json file and define an "executionEnvironment" that specifies the folder in which you intend to execute your script. For details, refer to [this documentation](https://github.com/microsoft/pyright/blob/master/docs/configuration.md#execution-environment-options).

For additional details on import resolution configuration, see [here](https://github.com/microsoft/pyright/blob/master/docs/import-resolution.md).
